src/AppInstallerCLICore/Workflows/ShellExecuteInstallerHandler.cpp

```diff
@@ -307,7 +347,142 @@ namespace AppInstaller::CLI::Workflow
             else
             {
                 context.Add<Execution::Data::OperationReturnCode>(uninstallResult.value());
+            }
+        }
+    }
+
+#ifndef AICLI_DISABLE_TEST_HOOKS
+    std::optional<DWORD> s_EnableWindowsFeatureResult_Override{};

```

Comment:
There is another test hook that also passes an optional value like so:
`void TestHook_SetPinningIndex_Override(std::optional<std::filesystem::path>&& indexPath);`

Since I am also passing an optional DWORD value, I just did something similar to that.